invenio_oauthclient/handlers.py

```diff
@@ -350,6 +346,14 @@ def signup_handler(remote, *args, **kwargs):
                 account_setup=account_setup
             )
 
+        # Authenticate the user
+        if not oauth_authenticate(remote.consumer_key, user,
+                                  require_existing_link=False,
+                                  remember=current_app.config[
+                                      'OAUTHCLIENT_REMOTE_APPS']
+                                  [remote.name].get('remember', False)):
+            return redirect(url_for("security.login"))
```

Comment:
Thanks. Can you also include this description as comment inside the if statement?